apps/gateway/src/middlewares/provider.ts

```diff
@@ -61,20 +167,12 @@ export const supportedOrThrow = (id: string) => {
   }
 };
 
-const chatOrThrow = (id: string): LanguageModel => {
-  supportedOrThrow(id);
-  if (isEmbedding(id)) {
```

Comment:
Don't you still need this?

Reply:
Since this is an external facing API, I want to ensure the error message is clear enough. Is it?